test/io/misc/character-entities.cjs

```diff
@@ -0,0 +1 @@
+module.exports = require('character-entities')
```

Comment:
I do think it’s a bug in Node that it can’t require clear CJS (as the package has no `type:`).
They do seem to have a [JSON loader](https://github.com/nodejs/node/blob/2d83e743d934738a3ad26ed587eb7cfd6ca46081/lib/internal/modules/esm/translators.js#L273), so I’m wondering why it’s not used here.

Reply:
https://nodejs.org/api/esm.html#esm_experimental_json_modules

Reply:
I think this is different: you can import CJS from MJS. CJS should then be able to work as it used to, no?
Node knows this module is CJS, and it can require JSON files in CJS, but it seems to break on json-only modules

Reply:
I hope node team will backport stabilized json modules to node 12

Reply:
Is this something that should be raised with them perhaps?

Reply:
We went into it in depth on twitter but TL;DR the current experimental modules will not likely stabilize in their current form as JSON modules are being standardized by TC39 right now using import assertions due to security needs of browsers

https://github.com/tc39/proposal-json-modules